api/v1alpha2/common_types.go

```diff
@@ -29,6 +29,14 @@ const (
 	MachineInternalIP  MachineAddressType = "InternalIP"
 	MachineExternalDNS MachineAddressType = "ExternalDNS"
 	MachineInternalDNS MachineAddressType = "InternalDNS"
+
+	// The capability strings should be all lower case so we can
+	// ToLower() any user input.
+
+	InfraCapbilityPowerOn  = "poweron"
```

Comment:
As the comment suggestions, IMO, we leave this all lower case, so if in the future we do something like

```
spec:
...
  powerState: poweroff
...
```
or whatever we come up with, we can do a ToLower() on whatever value the user provides.  Otherwise we could enforce strict case matching on the terms and do PowerOn or whatever instead.